README.md

```diff
@@ -1,4 +1,14 @@
-# Gov Proposal Builder
+[![Netlify Status](https://api.netlify.com/api/v1/badges/1fe8896d-c11b-42cc-badb-6fb24f68318b/deploy-status)](https://app.netlify.com/sites/shimmering-cactus-88287f/deploys)
+
+
+# ⚖️ Gov Proposal Builder
+
+[Live Demo](https://agoric-gov-proposal-hub.netlify.app)
+
+
+## Disclaimer
+
+The software is provided "as is", without warranty of any kind, express or implied, including but not limited to the warranties of merchantability, fitness for a particular purpose and noninfringement. In no event shall the authors or copyright holders be liable for any claim, damages or other liability, whether in an action of contract, tort or otherwise, arising from, out of or in connection with the software or the use or other dealings in the software.
```

Comment:
I'm curious where that legalese came from. Is the source willing to be attributed?

Reply:
This is from the MIT license. And that is the plan, hopefully.